lib/rspec/rails/matchers/active_job.rb

```diff
@@ -67,7 +67,14 @@ def thrice
           end
 
           def failure_message
-            "expected to enqueue #{base_message}"
+            "expected to enqueue #{base_message}".tap do |msg|
+              if @unmatching_jobs.any?
+                msg << "\nUnmatched enqueued jobs:"
+                @unmatching_jobs.each do |job|
+                  msg << "\n* #{base_job_message(job)}"
```

Comment:
We don't use a '*' elsewhere when indenting, maybe for consistency just go with 2 spaces?